components/new-contribution-flow/StepProfileGuestForm.js

```diff
@@ -0,0 +1,147 @@
+            {inputProps => (
+              <StyledInput
+                {...inputProps}
+                defaultValue={data?.email}
+                placeholder="i.e. tanderson@thematrix.com"
+                type="email"
+                onChange={e => onChange({ stepProfile: { ...data, [e.target.name]: e.target.value } })}
+              />
+            )}
+          </StyledInputField>
+        </Box>
+      </Flex>
+      {amount && amount >= 500000 && (
+        <Flex justifyContent="space-between">
+          <Box width={1 / 2} mb={3} mr={1}>
+            <StyledInputField
+              label={<FormattedMessage id="ExpenseForm.AddressLabel" defaultMessage="Physical address" />}
+              htmlFor="address"
+              required
+            >
+              {inputProps => (
+                <StyledTextarea
+                  {...inputProps}
+                  defaultValue={data?.address}
+                  placeholder="160 Zion Ln.&#13;&#10;Temecula, CA&#13;&#10;90210"
+                  width="100%"
+                  height="150px"
+                  maxWidth={350}
+                  fontSize="LeadCaption"
+                  onChange={e => onChange({ stepProfile: { ...data, [e.target.name]: e.target.value } })}
+                />
+              )}
+            </StyledInputField>
+          </Box>
+          <Box width={1 / 2} mb={3} ml={1}>
+            <StyledInputField
+              name="country"
+              label={<FormattedMessage id="ExpenseForm.ChooseCountry" defaultMessage="Choose country" />}
+              htmlFor="country"
+              required
+            >
+              {inputProps => (
+                <InputTypeCountry
+                  {...inputProps}
+                  locale={locale}
+                  onChange={value => {
+                    setLocale(value);
+                    onChange({ stepProfile: { ...data, country: value } });
+                  }}
+                  value={data?.country}
+                />
+              )}
+            </StyledInputField>
+          </Box>
+        </Flex>
+      )}
+      <P fontSize="SmallCaption" color="black.600">
+        <FormattedMessage
+          id="ContributionFlow.PublicContribution"
+          defaultMessage="Your name and contribution will be public."
+        />
+      </P>
+      <StepProfileInfoMessage amount={amount} />
+      {interval && (
+        <P color="black.500" fontSize="Caption" my={3} data-cy="join-conditions">
+          <FormattedMessage
+            id="SignIn.legal"
+            defaultMessage="By joining, you agree to our <tos-link>Terms of Service</tos-link> and <privacy-policy-link>Privacy Policy</privacy-policy-link>."
+            values={{
+              // eslint-disable-next-line react/display-name
+              'tos-link': msg => <Link route="/tos">{msg}</Link>,
+              // eslint-disable-next-line react/display-name
+              'privacy-policy-link': msg => <Link route="/privacypolicy">{msg}</Link>,
```

Comment:
Thanks to `components/I18nFormatters.js`, there's a better alternative to do that:

```suggestion
              'tos-link': getI18nLink({ as: Link, route: '/tos' }),
              'privacy-policy-link': getI18nLink({ as: Link, route: '/privacypolicy' }),
```